cylc/uiserver/resolvers.py

```diff
@@ -332,7 +335,7 @@ async def play(
                     results[wflow] = 'started'
 
             except Exception as exc:
-                # oh noes, something went wrong, send back confirmation
+                log.exception(exc)
```

Comment:
Ditto (almost identical change made in #689)

Reply:
resolved